### PR TITLE
solution

### PR DESCRIPTION
**Pull request by mariiahorbova**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -2,6 +2,37 @@
...
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    for item_first, item_second in actors:
+        Actor.objects.create(
+            first_name=item_first,
+            last_name=item_second
+        )
```
Comment (Kirontiko) [inline main.py:25]: I think u can change it to generator, but it's up to you

Comment (Kirontiko) [reply main.py:25]: Also for best readability you can change item_first, item_second to first_name, last_name

File: main.py [hunk 2]
```diff
@@ -2,6 +2,37 @@
...
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
```
Comment (Kirontiko) [inline main.py:11]: U can transform here to comprehension also

**Review by Kirontiko — APPROVED** [review]

**Review by RedCatHot — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (YuriiKindrat) [reply main.py:25]: @Kirontiko is right, it is better to name variables as first_name, last_name for better readability

**Pull request closed without merging**